crates/nu-protocol/src/ast/expr.rs

```diff
@@ -19,7 +19,7 @@ pub enum Expr {
     Var(VarId),
     VarDecl(VarId),
     Call(Box<Call>),
-    ExternalCall(Box<Expression>, Vec<Expression>),
+    ExternalCall(Box<Expression>, Vec<Expression>, bool), // head, args, is_subexpression
```

Comment:
Would it alternatively make sense to have two different external call variants? (Again not sure if this solves the problem in a simpler way or if it is more complex to disambiguate at parse time)

Reply:
Have thought about it, but finally I think it's not necessary, and we may leads to duplicate code inside `ExternalCall` and `SubExternalCall`(imagine that's the variant name)

Reply:
Yeah that's more a question of what reads more understandable or is easier to document with the code. We seem to need to carry the information anyways.